test_runner/fixtures/utils.py

```diff
@@ -37,6 +37,58 @@
 
 
 Fn = TypeVar("Fn", bound=Callable[..., Any])
+COMPONENT_BINARIES = {
+    "storage_controller": ["storage_controller"],
+    "storage_broker": ["storage_broker"],
+    "compute": ["compute_ctl"],
+    "safekeeper": ["safekeeper"],
+    "pageserver": ["pageserver", "pagectl"],
+}
+
+VERSIONS_COMBINATIONS = [
```

Comment:
Let's use tuples instead of lists for `COMPONENT_BINARIES` and `VERSIONS_COMBINATIONS`

Reply:
done, thanks